components/board/genericlinux/board.go

```diff
@@ -224,9 +223,9 @@ func (b *SysfsBoard) reconfigureAnalogs(ctx context.Context, newConf *Config) er
 // This helper function is used while reconfiguring digital interrupts. It finds the new config (if
 // any) for a pre-existing digital interrupt.
 func findNewDigIntConfig(
-	interrupt *digitalInterrupt, newConf *Config, logger golog.Logger,
+	interrupt *digitalInterrupt, confs []board.DigitalInterruptConfig, logger golog.Logger,
```

Comment:
This is a change that should have happened months ago, if I'd have noticed at the time. We don't need the entire config in here, just the subsection about digital interrupts.